doc/release-notes.md

```diff
@@ -24,6 +24,10 @@ RPC Changes
 - Several `z_sendmany` failures have moved from synchronous to asynchronous, so
   while you should already be checking the async operation status, there are now
   more cases that may trigger failure at that stage.
+- The `AllowRevealedRecipients` privacy policy is now required in order to choose a
+  transparent change address for a transaction (and thus `AllowFullyTransparent`
+  is required for the transaction, since transparent change implies a
+  transparent sender).
```

Comment:
```suggestion
- The `AllowRevealedRecipients` privacy policy is now required in order to choose a
  transparent change address for a transaction when the wallet is unable to construct the
  transaction without selecting funds from the transparent pool. The impact of this change 
  is that for such transactions, the user must specify `AllowFullyTransparent` since the 
  presence of transparent change implies a transparent sender.
```

Reply:
How about
```suggestion
- The `AllowRevealedRecipients` privacy policy is now required in order to
  choose a transparent change address for a transaction (which only occurs when
  the wallet is unable to construct the transaction without selecting funds from
  the transparent pool). The impact of this change is that for such
  transactions, the user must specify `AllowFullyTransparent`.
```